### PR TITLE
Adding border to tabs

### LINKED ISSUE #6286: [CAL-686] /bookings - tabs don't align with designs and spacing between tabs & heading is too tight
Should be `20px` / `...-5` 

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-686](https://linear.app/calcom/issue/CAL-686/bookings-tabs-dont-align-with-designs-and-spacing-between-tabs-and)</sub>

### PR DESCRIPTION
**Pull request by sean-brydon**
Adds border to tab to match the design system: 
https://www.figma.com/file/ZJNAFHCAq6rnuGRgUFxjzQ/Cal.com-Teams?node-id=439%3A55821&t=gW0CNjRpXs3AVbrN-4

Mobile:
CleanShot 2023-01-05 at 12 21 01@2x

Desktop
CleanShot 2023-01-05 at 12 21 39@2x

### CODE REVIEW

File: packages/ui/v2/core/navigation/tabs/HorizontalTabs.tsx [hunk 1]
```diff
@@ -10,8 +10,8 @@ export interface NavTabProps {
 
 const HorizontalTabs = function ({ tabs, linkProps, actions, ...props }: NavTabProps) {
   return (
-    <div className="-mx-6 mb-2 w-[calc(100%+40px)]">
-      <nav className="no-scrollbar flex space-x-1 overflow-scroll px-6" aria-label="Tabs" {...props}>
+    <div className="mb-2 max-w-[calc(100%+40px)]">
```
Comment (sean-brydon) [inline packages/ui/v2/core/navigation/tabs/HorizontalTabs.tsx:13]: This element has just been huge for no reason before now

**Comment by sean-brydon** [comment]
Also fixes the misalignment you spotted @PeerRich

**Review by CarinaWolli — APPROVED** [review]
LGTM 👍

**Comment by Jaibles** [comment]
@sean-brydon Looks great! 

One small thing, can we ensure that margin between the tabs & the empty state component and/or any content below is `20px` `mb-5`?

**Pull request merged**